x/storage/keeper/rewards.go

```diff
@@ -153,13 +153,13 @@ func (k Keeper) manageDealReward(ctx sdk.Context, deal types.ActiveDeals, networ
 		ctx.Logger().Error(e.Error())
 		return e
 	}
-	misses.SubRaw(1)
+	updatedMisses := misses.SubRaw(1)
```

Comment:
SubRaw(1) does not directly modify its receiver. It functions to return a new Int that should be assigned to a value. We could change this line to become 'misses = misses.SubRaw(1)', but I opine it's clearer to assign the result of SubRaw(1) to a new variable called 'updatedMisses'.

Reply:
Yeah this is my bad, excellent work